assets/i18n/en.json

```diff
@@ -653,41 +663,56 @@
     "I18N_LEARNER_DASHBOARD_VIEW_SUGGESTION": "View Suggestion",
     "I18N_LEARNER_GROUPS_SECTION_TITLE": "Your Learner Groups",
     "I18N_LEARNER_GROUP_ADD_GROUP_DETAILS": "Add Group Details",
-    "I18N_LEARNER_GROUP_ADD_NEW_LEARNER": "New Learner",
     "I18N_LEARNER_GROUP_ADD_NEW_SYLLABUS_ITEMS": "Add New Syllabus Items",
     "I18N_LEARNER_GROUP_ADD_SYLLABUS_ITEMS": "Add Syllabus Items",
     "I18N_LEARNER_GROUP_ADD_TO_SYLLABUS": "Add to syllabus",
     "I18N_LEARNER_GROUP_ASSIGNED_SKILLS": "{skillsCount, plural, =1{Assigned Skill} other{Assigned Skills}}",
     "I18N_LEARNER_GROUP_ASSIGNED_STORIES": "{storiesCount, plural, =1{Assigned Story} other{Assigned Stories}}",
+    "I18N_LEARNER_GROUP_ASSIGNED_SYLLABUS_TAB": "Assigned Syllabus",
     "I18N_LEARNER_GROUP_BACK_TO_ALL_LEARNERS_PROGRESS": "Back To All Learners' Progress",
     "I18N_LEARNER_GROUP_BACK_TO_SYLLABUS": "Back To Syllabus",
     "I18N_LEARNER_GROUP_CREATED_TITLE": "Your group <b><[groupName]></b> has been created.",
     "I18N_LEARNER_GROUP_CREATION_NEXT_BUTTON_TEXT": "Next",
     "I18N_LEARNER_GROUP_CREATION_PREV_BUTTON_TEXT": "Previous Step",
     "I18N_LEARNER_GROUP_DETAILS_GROUP_DESCRIPTION": "Group Description (Describe the group objectives in 2-4 lines)",
     "I18N_LEARNER_GROUP_DETAILS_GROUP_TITLE": "Group Title",
+    "I18N_LEARNER_GROUP_DETAILS_MODAL_DESCRIPTION": "Group Description",
+    "I18N_LEARNER_GROUP_DETAILS_TITLE": "Details",
+    "I18N_LEARNER_GROUP_FACILITATOR_MODAL_TEXT": "Facilitator",
     "I18N_LEARNER_GROUP_GROUP_DETAILS_SECTION": "Group Details",
+    "I18N_LEARNER_GROUP_INVITATION_MODAL_HEADER": "Learner group invitation",
     "I18N_LEARNER_GROUP_INVITE_LEARNERS": "Invite Learners",
     "I18N_LEARNER_GROUP_INVITE_LEARNERS_BY_USERNAME": "Invite learners through username",
     "I18N_LEARNER_GROUP_INVITE_LEARNERS_PLACEHOLDER_TEXT": "Add username of the learner to invite and press enter",
     "I18N_LEARNER_GROUP_INVITE_LIST_TEXT": "Invite List",
     "I18N_LEARNER_GROUP_ITEM_ADDED_TO_SYLLABUS": "Added",
     "I18N_LEARNER_GROUP_ITEM_ALREADY_ADDED_TO_SYLLABUS": "Already In Syllabus",
+    "I18N_LEARNER_GROUP_JOINING_MESSAGE": "You are about to join \"<[groupName]>\"",
     "I18N_LEARNER_GROUP_LEARNERS": "{learnersCount, plural, =1{LEARNER} other{LEARNERS}}",
+    "I18N_LEARNER_GROUP_LEARNERS_MODAL_TEXT": "Group Learners",
     "I18N_LEARNER_GROUP_LEARNERS_PROGRESS_TAB": "Learners' Progress",
     "I18N_LEARNER_GROUP_LEARNERS_SECTION": "{learnersCount, plural, =1{Learner} other{Learners}}",
     "I18N_LEARNER_GROUP_MINIMUM_SYLLABUS_ITEMS_INFO": "You need to add at least one syllabus item(Skill/Story) in order to make a group.",
+    "I18N_LEARNER_GROUP_NO_INVITATIONS": "You do not have any pending invitations.",
     "I18N_LEARNER_GROUP_NO_ITEMS_ADDED": "You haven't added new items yet.",
     "I18N_LEARNER_GROUP_NO_LEARNERS_HAVE_JOINED": "No learners have joined the group yet.",
```

Comment:
There is no one in this group. How about inviting a learner?

Reply:
done